src/models/userDao.js

```diff
@@ -0,0 +1,21 @@
+const { dataSource } = require('./dataSource');
+
+const userGetTickets = async (userId) => {
```

Comment:
```suggestion
const getTicketInfoByUserId = async (userId) => {
```

- 함수 이름은 동사로 시작해야 하는 것이 좋습니다. 이 함수가 어떤 동작을 수행하는지 명확히 파악하기 위해서입니다.
- Dao에서 선언되는 함수는 특히나 'Create, Read(Get), update, delete' 로 시작하는 함수 외에는 거의 없다고 보시면 됩니다. Entity를 곧바로 관리하는 함수이기 때문입니다. 어떤 Entity(Table 이름) 를 어떻게 (CRUD 중 어떤 동작으로 조작하는지) 가공하는지를 함수명에서 드러내시는 것이 좋습니다 :)

추가적으로 이 API는 사실 `user` 데이터를 중점적으로 다루는 API라고 보기는 어렵고, `ticket`데이터를 중점적으로 다루는 데이터네요 :) 따라서, 
- userController -> eventController 또는 ticketController
- userService -> eventService 또는 ticketService
- userDao -> eventDao 또는 ticketDao 로 변경해주세요